core/setup/kubernetes.go

```diff
@@ -47,47 +52,71 @@ func kubernetesParse(c *Controller) (kubernetes.Kubernetes, error) {
      * API calls may be desirable.
      */ 
 
-	k8s := kubernetes.Kubernetes{
+    k8s := kubernetes.Kubernetes{
         Proxy:      proxy.New([]string{}),
-		Ctx:        context.Background(),
-//		Inflight:   &singleflight.Group{},
+        Ctx:        context.Background(),
+//      Inflight:   &singleflight.Group{},
```

Comment:
Don't know yet. I suspect that I'll need to serialize k8s API calls to a single in-flight call.